source/content/guides/new-relic/06-troubleshoot-new-relic.md

```diff
@@ -77,6 +77,10 @@ You must manually [remove a deleted Multidev environment in New Relic](https://d
 
 1. Navigate to the bottom of the page, click the **Delete application** button, then click the **Confirmation** button when prompted.
 
+## Enable Browser Monitoring for Drupal 10.2+
+
+With the release of Drupal 10.2, [a change to http headers in Drupal core](https://www.drupal.org/node/3298551) prevents automatic injection of New Relic Browser Monitoring code. To continue using this feature, manually insert the tracking script into your templates. [Visit the New Relic documentation for an example configuration](https://docs.newrelic.com/docs/apm/agents/php-agent/features/browser-monitoring-php-agent/#manual_drupal).
```

Comment:
This recommendation seems like an un-Pantheon recommendation. Couldn't we do this for our customers in a module or the upstream?

Reply:
The New Relic docs also aren't very helpful. They include copy-paste-able examples for Drupal 7 but not Drupal 10

Reply:
We'll explore platform solutions in the future, but would be best for us to have this addressed in docs in the interim.

Reply:
Sounds good. Do you have a copy/pasteable example from a Drupal site that has made this change? Have any customers had success with the manual route?

Reply:
Discussed in slack that we will follow-up separately in a new PR with an SME to solve for a copy/pasteable solution